test/SFSmartStoreTestSuite.js

```diff
@@ -88,6 +88,86 @@ SmartStoreTestSuite.prototype.testGetDatabaseSize  = function() {
 };
 
 /** 
+ * TEST global store vs regular store with registerSoup / soupExists / removeSoup
+ */
+SmartStoreTestSuite.prototype.testRegisterRemoveSoupGlobalStore = function()  {
```

Comment:
The only test at this point exercising global smartstore - but more should be added